### PR TITLE
pvr.dvblink version 3.3.5 (krypton)

### PR DESCRIPTION
**Pull request by dvblogic**
Fixed: Crash on startup with search timers
Fixed: Incorrect disk space display if disk size is more than 2TB

### CODE REVIEW

File: src/DVBLinkClient.cpp [hunk 1]
```diff
@@ -736,11 +736,11 @@ int DVBLinkClient::GetSchedules(ADDON_HANDLE handle, const RecordingList& record
     timer.iMarginEnd = bp_schedules[i]->MarginAfter / 60;
     strncpy(timer.strEpgSearchString, bp_schedules[i]->GetKeyphrase().c_str(), sizeof(timer.strEpgSearchString) - 1);
 
-    if (schedule_to_timer_map.find(epg_schedules[i]->GetID()) != schedule_to_timer_map.end() &&
-      schedule_to_timer_map[epg_schedules[i]->GetID()].size() > 0)
+    if (schedule_to_timer_map.find(bp_schedules[i]->GetID()) != schedule_to_timer_map.end() &&
+      schedule_to_timer_map[bp_schedules[i]->GetID()].size() > 0)
```
Comment (fritsch) [inline src/DVBLinkClient.cpp]: might want to use !empty() ?

**Comment by dvblogic** [comment]
Done.
Can you, please, also merge the pull request for Jarvis branch?

**Comment by fritsch** [comment]
This is something for @ksooo I was just interested in looking at the code.

**Comment by dvblogic** [comment]
@fritsch 
Is there any way to contact @ksooo or anyone else, who can merge those pull requests?..

**Comment by ksooo** [comment]
@dvblogic i have no good feeling to merge this if travis-ci check fails. Any chance that you fix that?

**Comment by dvblogic** [comment]
@ksooo Please, have a look at the Travis log. It is some kind of build misconfiguration issue, which is outside of my competence.

**Comment by ksooo** [comment]
i'm also not an expert on this, but maybe https://github.com/kodi-pvr/pvr.dvblink/blob/master/.travis.yml needs to be fixed? 
This file looks little different for pvr.hts, for instance. https://github.com/kodi-pvr/pvr.hts/blob/master/.travis.yml
Maybe changing pvr.dvblogic's .travis.yml to like the one for pvr.hts looks like already does the trick?

**Comment by dvblogic** [comment]
@ksooo can someone from kodi team fix this issue, please? I really do not want to mess around with build variables, not knowing what they do. And, more important, not been able to test.

**Comment by ksooo** [comment]
@hudokkow maybe you can help with the travis problem?

**Comment by hudokkow** [comment]
@ksooo nothing really different between the 2 apart from hts not depending on tinyxml2 and wrongly calling <code>script: make</code> twice.

Apart from that, build error is self-explanatory. We require CMake 3.1 and xcode6.1 machine only provides 3.0.2. I'll bug travis folks about this.

In the meantime, if you guys are happy with the code changes it should be safe to merge. And if a tiny red cross is something you really don't like, comment out https://github.com/kodi-pvr/pvr.dvblink/blob/master/.travis.yml#L22-L23 :stuck_out_tongue_winking_eye:

**Comment by ksooo** [comment]
@hudokkow thanks for the explanation. will merge now

**Pull request merged**